scss/tabstrip/_theme.scss

```diff
@@ -62,30 +62,30 @@ $tabstrip-content-border: $panel-bg !default;
     .k-tabstrip-top {
         > .k-tabstrip-items {
             .k-item.k-state-active {
-                border-bottom-color: $tabstrip-item-selected-bg;
+                border-bottom-color: $tabstrip-item-selected-border;
```

Comment:
that needs to be `-bg`, as well as the rest of the instances, so we can get the "enclosure" like appearance.